### DISCUSSION TITLE
Personal dictionary words not suggested after Emacs restart with Hunspell backend

### DISCUSSION THREAD

**Discussion by AledavR** [minad/jinx#217] [Q&A]
Hi, I'm using this package and have noticed an issue. When a word is added to the personal dictionary, it is suggested as an option for future misspellings, but only during the current Emacs session. If I restart Emacs, the word is recognized as correct, but if I type another misspelled word that is close to the one in the personal dictionary, it is no longer offered as an option.

I'm using the Hunspell backend with the default configuration from the README, and the personal dictionary is located at ~/.config/enchant/en_US.dic. This behavior can be observed with the en_US and es_any Hunspell packages on Arch Linux.

**Comment by minad** [comment]
I see this problem. It looks to me like an issue in Enchant. I asked @rrthomas upstream about this: https://github.com/rrthomas/enchant/issues/403.

**Reply by minad** [reply]
Oh, now I saw this in the Enchant NEWS file:

~~~
First, Enchant’s mechanism for generating suggestions from personal
wordlists is removed. This mechanism was complicated, would generate
suggestions in a different way from each provider, and was unlikely to work
well for many languages. Instead, support has been improved for supplying
words from the user’s personal wordlist to the provider, so that it can
include them in suggestions. This is supported by the Aspell and Hunspell
backends.
~~~

I am not sure if the personal wordlist is supplied automatically to Hunspell or if this doesn't work for some reason when generating suggestions.

**Comment by minad** [comment]
Closing. Please follow https://github.com/rrthomas/enchant/issues/403.
